### PR TITLE
Changed regionid replace to the cat label function

### PR DESCRIPTION
**Pull request by garypolzin**
# Developer checklist
- [X] Unit test

### CODE REVIEW

**Review by MarionJHolloway — COMMENTED** [review]
Is there a reason this doesn't use the categorical labels function we already built and tested?

**Comment by garypolzin** [comment]
> Is there a reason this doesn't use the categorical labels function we already built and tested?

probably laziness! Changed it to the cat labels function

File: jobs/reconciliation.py [hunk 1]
```diff
@@ -122,7 +124,12 @@ def prepare_latest_cleaned_ascwds_workforce_data(
     df = utils.filter_df_to_maximum_value_in_column(
         ascwds_workplace_df, AWPClean.ascwds_workplace_import_date
     )
-    df = replace_region_id_labels_for_reconciliation(df)
+    df = cUtils.apply_categorical_labels(
+        df,
+        labels_dict,
```
Comment (MarionJHolloway) [inline jobs/reconciliation.py:129]: Might be worth giving this a more specific name? E.g. reconciliation_labels_dict. I'm being quite picky here though!

**Review by MarionJHolloway — APPROVED** [review]
Looks good!

**Pull request merged**